examples/vault-cluster-private/main.tf

```diff
@@ -68,6 +68,23 @@ data "template_file" "user_data_vault_cluster" {
   }
 }
 
+# ---------------------------------------------------------------------------------------------------------------------
+# PERMIT CONSUL SPECIFIC TRAFFIC IN VAULT CLUSTER
+# To allow our Vault servers consul agents to communicate with other consul agents and participate in the LAN gossip,
+# we open up the consul specific protocols and ports for consul traffic
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "security_group_rules" {
+  source = "github.com/hashicorp/terraform-aws-consul.git//modules/consul-client-security-group-rules?ref=v0.3.1"
```

Comment:
This version number will need to be bumped once your Consul PR is merged

Reply:
@brikis98 thanks for merging the terraform-aws-consul PR ... do you think I should also update the S3 example to incorporate this change as well? And just a small guidance, should I also update the version number in the `consul-cluster` module?

Reply:
> do you think I should also update the S3 example to incorporate this change as well

If it's not too much of a pain, it would be great to update all the examples in this repo: the `main.tf` in the root, `examples/vault-cluster-private`, and `examples/vault-s3-backend`. 

> And just a small guidance, should I also update the version number in the consul-cluster module?

I think it'll be confusing to be using different terraform-aws-consul versions in different places, so probably a global search & replace to update to the latest version is the best option.